src/plugin.ts

```diff
@@ -18,16 +18,32 @@ async function main() {
   const typeMap = createTypeMap(request, options);
   const utils = makeUtils(options);
   const ctx: Context = { typeMap, options, utils };
+  let filesToGenerate;
+
+  if (options.emitImportedFiles) {
+    const fileSet = new Set();
+    const addFilesDeep = (filenames: string[]) => {
```

Comment:
Wdyt of renaming this to `addFilesUnlessAliased`? Just to highlight that we're filtering those out.